svc-aggregation/agcommon/common.go

```diff
@@ -12,7 +12,6 @@
 //License for the specific language governing permissions and limitations
 // under the License.
 
-// Package agcommon ...
```

Comment:
Please run the below and update the same
1.	go fmt ./…
2.	go vet ./…
3.	golint ./…